provisioner/provision_lab.yml

```diff
@@ -166,6 +166,38 @@
         - workshop_type == "networking"
         - ansible_network_os == "ios" or ansible_network_os == "eos"
 
+- name: INSTALL AND CONFIGURE SPLUNK ENTERPRISE
+  hosts: splunk
+  become: true
+  gather_facts: no
+  roles:
+    - role: splunk_enterprise
+      when:
+        - workshop_type == "security_automation"
+
+- name: INSTALL AND CONFIGURE SNORT
+  hosts: snort
+  become: true
+  vars:
+    ids_config_snort_version: '2.9.13'
```

Comment:
How about putting those variables in additional group_vars? Best practices recommend to not put variables into playbooks IIRC.

Reply:
I'll move them because that's definitely cleaner, but I'm curious what Best Practices guide you're referencing because there's no mention of it that I'm seeing in the [core docs](https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html#variables-and-vaults).

Reply:
Yeah, sorry for that, I might have dreamed that. So skip this comment.